guides/git-guidelines.md

```diff
@@ -1,60 +1,64 @@
+* **1.** [Commit Message Guidelines](#CommitMessageGuidelines)
+* **1.1** [Message Format](#MessageFormat)
+* **1.1.1** [Message Header Type](#HeaderType)
+* **1.1.2** [Message Header Scope](#HeaderScope)
+* **1.1.3** [Message Samples](#CommitMessageSamples)
+* **2.** [Git Branches Naming](#GitBranchesNaming)
+* **3.** [Git Workflow](#GitWorkflow)
+* **4.** [Credits](#Credits)
 
-## Commit Message Guidelines
+Please, notice that at MarsBased we work with a large variety of clients. There could be clients that follow their own guidelines. You can always suggest improvements over their guidelines but there will be cases where it won't be possible to use ours.
 
-We have very precise rules over how our git commit messages can be formatted.
-This leads to more readable messages that are easy to follow when looking through the project history.
+## <a name='CommitMessageGuidelines'></a>Commit Message Guidelines
 
-### Commit Message Format
+We have very precise rules over how our git commit messages can be formatted. This leads to more readable messages that are easy to follow when looking through the project history.
+
+### <a name='MessageFormat'></a>Commit Message Format
 
 Each commit message consists of a header, a body and a footer. The header has a special format that includes a type, a scope and a subject:
 
 ```
 <type>(<scope>): <subject>
-<BLANK LINE>
+-BLANK LINE-
 <description>
-<BLANK LINE>
+-BLANK LINE-
 <footer>
 ```
 
-The header is mandatory and the scope of the header is **optional**.
+The header is mandatory but you can leave his scope out as it is **optional**.
 
 The maximum length of the header must be 72 characters and any other line of the commit message cannot be longer 100 characters. This allows the message to be easier to read on GitHub as well as in various git tools.
 The language used in the commit messages is English. If the client wants to have access to the commit history for documentation purposes and they don't understand English, other languages can be used instead.
 
-The `<footer>` should contain a [closing reference to a github issue](https://help.github.com/en/github/managing-your-work-on-github/closing-issues-using-keywords),
-a Trello card link, a JIRA issue ID or link. If there is no project management tool that can be referenced or the issue was not created for this specific commit.
+The `<footer>` should contain a closing reference to a [github issue](https://help.github.com/en/github/managing-your-work-on-github/closing-issues-using-keywords), a Trello card link, a JIRA issue ID or link. If there is no GitHub issue or project management tool reference for this specific commit just leave it blank.
 
-#### Type
+#### <a name='HeaderType'></a>Type
 
 Choose the one that best fits the task:
 
 - __fix:__ Represents a bug fix for your application.
-- __feat:__ Adds a new feature to your application or library.
+- __feature:__ Adds a new feature to your application or library.
```

Comment:
I find kind of weird include the `:` inside the emphasis. OCD you say? 😄 
```suggestion
- __feature__: Adds a new feature to your application or library.
```

Reply:
me too! :D